components/infobox/commons/infobox_team.lua

```diff
@@ -48,18 +49,27 @@ end
 function Team:createInfobox()
 	local infobox = self.infobox
 	local args = self.args
-	-- Need links in LPDB, so declare them outside of display code
+
+	--- Transform data
+	-- Links
 	local links = Links.transform(args)
 
+	-- Team Information
 	local team = args.teamtemplate or self.pagename
 	self.teamTemplate = mw.ext.TeamTemplate.teamexists(team) and mw.ext.TeamTemplate.raw(team) or {}
```

Comment:
```suggestion
	self.teamTemplate = mw.ext.TeamTemplate.raw(team) or {}
```
would work too, since `mw.ext.TeamTemplate.raw` returns nil if the tt does not exist